tests/conftest.py

```diff
@@ -6,8 +6,8 @@
 import ape
 
 # NOTE: Ensure that we don't use local paths for these
-ape.config.DATA_FOLDER = Path(mkdtemp())
-ape.config.PROJECT_FOLDER = Path(mkdtemp())
+ape.config.DATA_FOLDER = Path(mkdtemp()).resolve()
```

Comment:
ran into crazy weirdness because this is a symlink but resolving the symlink to the real path helped